packages/embarkjs/.npmrc

```diff
@@ -0,0 +1,4 @@
+engine-strict = true
```

Comment:
Same here, that can probably be at the root?

Reply:
Unfortunately, `.npmrc` must be a sibling to `package.json`, can't be cleaned up as I did for `.yarnrc` in #1298.